plugins/trace_api_plugin/trace_api_plugin.cpp

```diff
@@ -111,13 +114,24 @@ struct trace_api_common_impl {
 
       slice_stride = options.at("trace-slice-stride").as<uint32_t>();
 
-      store = std::make_shared<store_provider>(trace_dir, slice_stride);
+      const char* trace_minimum_irreversible_history_blocks = "trace-minimum-irreversible-history-blocks";
+      const int32_t blocks = options.at(trace_minimum_irreversible_history_blocks).as<int32_t>();
+      EOS_ASSERT(blocks >= -1, chain::plugin_config_exception,
+                 "\"${blocks}\" must be greater to or equal to -1.",("blocks",trace_minimum_irreversible_history_blocks));
```

Comment:
I think `trace_minimum_irreversible_history_blocks` is the `const char *` and this assert message's value should be `blocks` right? 

or, is the message just meant to deduplicate the constant string?   maybe change the "key" to `${opt}` if so?

as is its confusing.

Reply:
I find the `const char *` of same name as string not helpful. I say just duplicate the string.

Reply:
ok, but I hate making 2 strings always match, let along 3.